WordPress/Classes/ViewRelated/AccountSettingsViewController.swift

```diff
@@ -3,12 +3,14 @@ import UIKit
 import RxSwift
 import WordPressComAnalytics
 
-func AccountSettingsViewController(account account: WPAccount) -> ImmuTableViewController {
-    let service = AccountSettingsService(userID: account.userID.integerValue, api: account.restApi)
+func AccountSettingsViewController(account account: WPAccount?) -> ImmuTableViewController {
+    let service = account.map({ account in
+        return AccountSettingsService(userID: account.userID.integerValue, api: account.restApi)
+    })
```

Comment:
Just asking, because `Map`, although a valid statement here, is mostly used for collections. But i think it's just a matter of taste. 

P.s: You can drop the parenthesis, if you opt for the `map` style. 

```
let service = account.map { account in
   return AccountSettingsService(userID: account.userID.integerValue, api: account.restApi)
}
```